### PR TITLE
data: geeky-sambhav

### PR DESCRIPTION
**Pull request by geeky-sambhav**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

You are doing a great job. I would like my Linkfree profile to be updated

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5656"></a>

### CODE REVIEW

File: geeky-sambhav.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	"type": "personal",
+	"tags": ["Open Source", "Javascript", "Collaboration", "ReactJS"],
+	"bio": "Frontend Developer,BibliophileVoyager,WellnessWizard",
+	"socials": [{
+			"icon": "FaTwitter",
+			"url": "https://twitter.com/Sambhav_Gandhi"
+		},
+		{
+			"icon": "FaGithub",
+			"url": "https://github.com/geeky_sambhav"
```
Comment (khushalarora) [inline geeky-sambhav.json:12]: This URL is broken, you can correct it :+1:

File: geeky-sambhav.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+		},
+		{
+			"icon": "FaLinkedin",
+			"url": "https://github.com/sambhav-gandhi"
```
Comment (khushalarora) [inline geeky-sambhav.json:16]: I think you want to use your LinkedIn url here :+1:

File: geeky-sambhav.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+		}
+	],
+	"links": [{
+			"name": "Follow me on GitHub",
+			"url": "https://github.com/geky-sambhav",
```
Comment (khushalarora) [inline geeky-sambhav.json:21]: Also, this URL is broken, you can correct it :+1:

File: geeky-sambhav.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+			"icon": "FaGithub"
+		},
+		{
+			"name": "Follow me on Linkedin",
+			"url": "https://www.linkedin.com/in/sambhav-gandhi",
```
Comment (khushalarora) [inline geeky-sambhav.json:26]: This URL is also broken :+1:

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @geeky-sambhav, thank you for contributing  :tada: 
Please rename your file and Add **`data/`** at the beginning :+1: 
It will look like **`data/geeky-sambhav.json`**

**Review by khushalarora — APPROVED** [review]
Looks good, thanks  :tada:

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 

<p align="center"></p>

**Pull request merged**